### PR TITLE
Docker setup, take 2

### PR DESCRIPTION
**Pull request by aelkiss**
Gets the docker stuff that is on `master` (but not `main`) and adds instructions for using this solr via docker from https://github.com/hathitrust/catalog

### CODE REVIEW

**Comment by aelkiss** [comment]
In retrospect, we shouldn't do this, and should rather provide:

1) a solr with sample records
2) instructions for starting this solr, indexing to it, and connecting to it from https://github.com/hathitrust/catalog, https://github.com/hathitrust/oai_solr, etc

**Comment by aelkiss** [comment]
I will rebase this on main.

**Comment by aelkiss** [comment]
@billdueber ready for re-review

**Review by billdueber — APPROVED** [review]
Looks good to me. I say we put it in as-is, and then remove the HLB stuff and move to jurby 9.3 pending a final double-check with the website folks to make sure they don't use it anywhere.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FROM jruby:9.2
```
[the remark this answers is not in the document]
Comment (aelkiss) [reply Dockerfile:1]: :+1:

**Pull request merged**